docs/compression_algorithms/CompressWeights.md

```diff
@@ -60,6 +60,64 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset)
 ```
 
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
```

Comment:
I would suggest to have a separate sample like we have for tiny-llama now.

Reply:
@MaximProshin can I do it in next PR ?

Reply:
yes, separate PR is ok